src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```diff
@@ -64,10 +64,10 @@
     <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(OS)' != 'Windows_NT' and '$(_targetOS)' == 'win'"
       Text="Cross-OS native compilation is not supported." />
 
-    <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(_targetArchitecture)' != 'x64' and '$(_targetArchitecture)' != 'x86' and '$(_targetArchitecture)' != 'arm64' and '$(_targetArchitecture)' != 'arm'"
+    <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(_targetArchitecture)' != 'x64' and '$(_targetArchitecture)' != 'x86' and '$(_targetArchitecture)' != 'arm64' and '$(_targetArchitecture)' != 'arm' and '$(_targetArchitecture)' != 'loongarch64'"
       Text="Native compilation does not support targeting $(RuntimeIdentifier) yet." />
 
-    <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(_hostArchitecture)' != 'x64' and '$(_hostArchitecture)' != 'x86' and '$(_hostArchitecture)' != 'arm64' and '$(_hostArchitecture)' != 'arm'"
+    <Error Condition="'$(DisableUnsupportedError)' != 'true' and '$(_hostArchitecture)' != 'x64' and '$(_hostArchitecture)' != 'x86' and '$(_hostArchitecture)' != 'arm64' and '$(_hostArchitecture)' != 'arm' and '$(_hostArchitecture)' != 'loongarch64'"
       Text="Native compilation can run on x64, x86, arm64 and arm hosts only." />
 
     <Error Condition="'$(IlcHostPackagePath)' == '' and '$(RuntimePackagePath)' != '' and ('$(_hostArchitecture)' == 'x64' or '$(_hostArchitecture)' == 'arm64')"
```

Comment:
Does it need adding loongarch64  here?

Reply:
@MichalStrehovsky may be able to confirm, but I think this condition can be simplified without the arch part. In other places, cross means 'host and target arch differ'. So the second part of this condition is not strictly necessary. I have some idea what it is trying to avoid, but even without this part of the condition is not going to degrade the user-experience.

Reply:
Yeah, I don't know what that condition is guarding against, I would just delete the `_hostArchitecture` part and see if it breaks anything.

Reply:
```suggestion
    <Error Condition="'$(IlcHostPackagePath)' == '' and '$(RuntimePackagePath)' != ''"
```